aten/src/ATen/core/dispatch/DispatchKeyExtractor.h

```diff
@@ -117,12 +117,12 @@ struct CAFFE2_API DispatchKeyExtractor final {
         }
       }
     }
-    return dispatchKeySetToDispatchKey_(backendsWithoutFallthrough, ks);
+    return dispatchKeySetToDispatchKey_(backendsWithoutFallthrough, ks.add(DispatchKey::BackendSelect));
```

Comment:
Weren't we going to put `BackendSelect` in the default key set? https://github.com/pytorch/pytorch/blob/98ab1877d3703c33413f2522082e055449a034a7/aten/src/ATen/core/dispatch/DispatchKeyExtractor.h#L17